Automattic/wp-calypso
# Launchpad: Add domain ssl processing notification

### Proposed Changes

* Show a notification message underneath the domains preview noting that it is in the process of being set up

### Screenshots

![Screen Shot 2022-10-21 at 12 20 06 AM](https://user-images.githubusercontent.com/5414230/197136710-f514d280-a197-4d79-bb7e-21148233005d.png)

### Testing Instructions



* Check out this PR
* Create new site through tailored flow https://wordpress.com/hp-2022-tailored-flows/
* Make sure to switch out wordpress.com with calypso.localhost:3000
* Create a custom domain for the site
* Choose, at minimum, a personal plan ( so that we can make a custom domain the primary domain )
* Complete steps until at the Launchpad screen
* Verify that there is a domain ssl processing notification until ssl is active on the custom domain

#### Pre-merge Checklist






Related to https://github.com/Automattic/wp-calypso/issues/68967
